pkg/registry/core/pod/strategy.go

```diff
@@ -212,7 +212,11 @@ func PodToSelectableFields(pod *api.Pod) fields.Set {
 	podSpecificFieldsSet["spec.schedulerName"] = string(pod.Spec.SchedulerName)
 	podSpecificFieldsSet["spec.serviceAccountName"] = string(pod.Spec.ServiceAccountName)
 	podSpecificFieldsSet["status.phase"] = string(pod.Status.Phase)
-	podSpecificFieldsSet["status.podIP"] = string(pod.Status.PodIP)
+	podIP := ""
+	if len(pod.Status.PodIPs) > 0 {
+		podIP = string(pod.Status.PodIPs[0].IP)
+	}
+	podSpecificFieldsSet["status.podIP"] = podIP
```

Comment:
For next stage we will need to add podIPs as a downward API value, and decide hopw to format it.

Reply:
Mayeb add a TODO here?

Reply:
added